include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (215)
+#define OPENTHREAD_API_VERSION (216)
```

Comment:
I don't see any change to the API. Can we avoid this change?

Reply:
I agree. I will revert back the version number bump.